x-pack/plugins/cloud_security_posture/public/pages/findings/findings_flyout/overview_tab.tsx

```diff
@@ -152,33 +161,34 @@ export const OverviewTab = ({ data }: { data: CspFinding }) => {
   );
 
   const accordions: Accordion[] = useMemo(
-    () => [
-      {
-        initialIsOpen: true,
-        title: i18n.translate('xpack.csp.findings.findingsFlyout.overviewTab.detailsTitle', {
-          defaultMessage: 'Details',
-        }),
-        id: 'detailsAccordion',
-        listItems: getDetailsList(data, discoverIndexLink),
-      },
-      {
-        initialIsOpen: true,
-        title: i18n.translate('xpack.csp.findings.findingsFlyout.overviewTab.remediationTitle', {
-          defaultMessage: 'Remediation',
-        }),
-        id: 'remediationAccordion',
-        listItems: getRemediationList(data.rule),
-      },
-      {
-        initialIsOpen: false,
-        title: i18n.translate(
-          'xpack.csp.findings.findingsFlyout.overviewTab.evidenceSourcesTitle',
-          { defaultMessage: 'Evidence' }
-        ),
-        id: 'evidenceAccordion',
-        listItems: getEvidenceList(data),
-      },
-    ],
+    () =>
+      [
+        {
+          initialIsOpen: true,
+          title: i18n.translate('xpack.csp.findings.findingsFlyout.overviewTab.detailsTitle', {
+            defaultMessage: 'Details',
+          }),
+          id: 'detailsAccordion',
+          listItems: getDetailsList(data, discoverIndexLink),
+        },
+        {
+          initialIsOpen: true,
+          title: i18n.translate('xpack.csp.findings.findingsFlyout.overviewTab.remediationTitle', {
+            defaultMessage: 'Remediation',
+          }),
+          id: 'remediationAccordion',
+          listItems: getRemediationList(data.rule),
+        },
+        INTERNAL_FEATURE_FLAGS.showFindingFlyoutEvidence && {
```

Comment:
not deleting yet because the reason we wanted to hide it is that we don't always have this data. should be fixed soon, or we can just display it only if we have the data. needs to be decided